src/layouts/DetailsLayout.tsx

```diff
@@ -0,0 +1,33 @@
+import Container from '@mui/material/Container';
+import Typography from '@mui/material/Typography';
+
+import AppBar from 'components/AppBar';
+import BackButton from 'components/BackButton';
+import Main from 'components/Main';
+import ToolbarSpace from 'components/ToolbarSpace';
+import Wrapper from 'components/Wrapper';
+
+type DetailsLayoutProps = {
+    children?: React.ReactNode;
+    title?: React.ReactNode;
+};
+
+const DetailsLayout: React.FC<DetailsLayoutProps> = ({ children, title }) => {
+    return (
+        <Wrapper>
+            <AppBar>
+                <BackButton href=".." tooltipTitle="Назад" />
```

Comment:
вроде всегда ведёт на главную
```suggestion
                <BackButton href="/" tooltipTitle="Назад" />
```

Reply:
Когда мы смотрим конкретную точку входа или поле, я думаю логично вернутся к их спискам.